src/blueapi/rest/handler.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from pathlib import Path
+from typing import Optional
+
+from blueapi.config import ApplicationConfig, AppSettings
+from blueapi.core import BlueskyContext
+from blueapi.messaging import StompMessagingTemplate
+from blueapi.messaging.base import MessagingTemplate
+from blueapi.utils import ConfigLoader
+from blueapi.worker.reworker import RunEngineWorker
+from blueapi.worker.worker import Worker
+
+settings = AppSettings()
+
+
+class Handler:
+    context: BlueskyContext
+    worker: Worker
+    config: ApplicationConfig
+    message_bus: MessagingTemplate
+
+    def __init__(self, config_path: Optional[Path]) -> None:
+        self.context = BlueskyContext()
+
+        loader = ConfigLoader(ApplicationConfig)
+        if config_path is not None:
+            loader.use_yaml_or_json_file(config_path)
+        self.config = loader.load()
+        logging.basicConfig(level=self.config.logging.level)
+
+        self.context.with_startup_script(self.config.env.startup_script)
+
+        self.worker = RunEngineWorker(self.context)
+        self.message_bus = StompMessagingTemplate.autoconfigured(self.config.stomp)
+
+    def start(self) -> None:
+        self.worker.start()
+
+        self.worker.data_events.subscribe(
+            lambda event, corr_id: self.message_bus.send(
+                "public.worker.event.data", event, None, corr_id
+            )
+        )
+        self.worker.progress_events.subscribe(
+            lambda event, corr_id: self.message_bus.send(
+                "public.worker.event.progress", event, None, corr_id
+            )
+        )
+
+        self.message_bus.connect()
+
+    def stop(self) -> None:
+        self.worker.stop()
+
+
+HANDLER: Optional[Handler] = None
```

Comment:
Just a thought, one alternative to a global variable is to use an `lru_cache`: https://cosmiccoding.com.au/tutorials/simple_singletons/
Probably out of scope but could be worth making a ticket, what do you think?

Reply:
Good idea; I don't think it's out of scope at all, It's part of the whole 'handler' business. I'd prefer to avoid global variables. Thanks for looking into this.

Reply:
The only thing I'm not sure of, is if this cache can be accessed by multiple sources at the same time. I want 3/4 simultaneous API calls to get the handler with no problem, which with a global variable will happen but I'm not sure about the cache. I'm assuming this is fine but I can't explain to myself why it's fine, and I haven't found good documentation explaining this case yet...

Reply:
^ It is thread safe so hopefully that's a good indication it's fine. I might write some tests for this.

Reply:
I *think* the multiple access case is fine but good to double check. There is still a single object stored in the cache that is accessed by all requests. However, one thing that has just occurred to me is that the handler won't be created until the first REST call. Not sure we want this, we ideally want it to happen on startup so we know quickly if there's a problem. Maybe we can call the `lru_cache`ed function as a startup task. That will create a handler and put it in the cache.